lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1689,7 +1689,7 @@ def test_export_val_data_with_internal(self):
         # Also verify the content of created transcript file.
         expected_transcript_content = File(open(expected_transcript_path)).read()
         transcript = get_video_transcript_data(video_id=self.descriptor.edx_video_id, language_code=language_code)
-        self.assertEqual(transcript['content'], expected_transcript_content)
+        self.assertEqual(transcript['content'].decode('utf-8'), expected_transcript_content)
```

Comment:
Should transcript content be bytes?  Perhaps we should update the function so that the content that comes out of this function is unicode by default?  Otherwise we'll have fixed the test but not the underlying code used in production.

Reply:
This comes from https://github.com/edx/edx-val. should we update this.

Reply:
It looks like it is directly the contest of a read over there and given that it generally is dealing with files and filesystem it looks like for now we should assume anything coming out of that library is bytes and do the conversion we need to do.  My concern is that this will fix the test but we should probably also update any places in the code where we grab the content of a transcript and make that unicode before we handle it in edx-platform code?